activiti-cloud-audit-service/activiti-cloud-services-audit-api/src/main/java/org/activiti/cloud/services/audit/api/assembler/EventRepresentationModelAssembler.java

```diff
@@ -16,22 +16,31 @@
 
 package org.activiti.cloud.services.audit.api.assembler;
 
-import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
-import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
-
 import org.activiti.cloud.api.model.shared.events.CloudRuntimeEvent;
 import org.activiti.cloud.services.audit.api.controllers.AuditEventsController;
 import org.activiti.cloud.services.audit.api.converters.CloudRuntimeEventType;
-import org.springframework.hateoas.Link;
 import org.springframework.hateoas.EntityModel;
+import org.springframework.hateoas.Link;
 import org.springframework.hateoas.server.RepresentationModelAssembler;
 
+import java.lang.reflect.Method;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+
 public class EventRepresentationModelAssembler implements RepresentationModelAssembler<CloudRuntimeEvent<?, CloudRuntimeEventType>, EntityModel<CloudRuntimeEvent<?, CloudRuntimeEventType>>> {
 
     @Override
     public EntityModel<CloudRuntimeEvent<?, CloudRuntimeEventType>> toModel(CloudRuntimeEvent<?, CloudRuntimeEventType> event) {
-        Link selfRel = linkTo(methodOn(AuditEventsController.class).findById(event.getId())).withSelfRel();
+        Link selfRel = linkTo(getFindByIdMethod(), event.getId()).withSelfRel();
```

Comment:
The problem is in the DummyInvocationUtils class used by methodOn that doesn't work well with interfaces. If I use a concrete implementation everything works as expected. Using reflection there is a difference in "affordances" field as you can see in images below.
<img width="386" alt="Screenshot 2020-05-11 at 09 38 17" src="https://user-images.githubusercontent.com/41215/81541558-2f259f80-936b-11ea-9bf8-62347ad66a4e.png">